app/controllers/topics.controller.js

```
@@ -42,11 +44,39 @@ exports.postCommentByArticleId = async (req, res, next) => {
     const {article_id} = req.params
     const {username, body} = req.body
 
+    if (!body) {
+        throw {
+            status: 400,
+            msg: 'Bad request!'
+        }
+    }
+
     try {
         await selectArticleById(article_id)
         const newComment = await insertCommentByArticleId(article_id, {username, body})
         res.status(201).send({comment: newComment})
     } catch (err) {
         next(err)
     }
+}
+
+exports.patchArticlebyId = async (req, res, next) => {
+    const {article_id} = req.params
+    const {inc_votes} = req.body
+
+    if (isNaN(inc_votes) || isNaN(article_id)) {
```

PSQL should throw an error for you if inc_votes is not a number 👍